packages/fxa-graphql-api/src/lib/server.ts

```diff
@@ -39,7 +40,7 @@ export function formatError(debug: boolean, logger: Logger, err: GraphQLError) {
  * Context available to resolvers
  */
 export type Context = {
-  authUser: string;
+  session: { uid: string; state: 'verified' | 'unverified' };
```

Comment:
could enum `'verified'`, `'unverified'` since we use them in a number of places? might be overkill though.

Reply:
if we add more states maybe